CMakeLists.txt

```diff
@@ -88,14 +89,15 @@ IF (NOT DYNAMIC_LINKING)
 ELSE (NOT DYNAMIC_LINKING)
 	IF(MINGW)
 		# Avoid depending on MinGW runtime DLLs
-		check_cxx_compiler_flag (-static-libgcc HAVE_STATIC_LIBGCC_FLAG)
-		IF(HAVE_STATIC_LIBGCC_FLAG)
-			SET(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -static-libgcc")
-		ENDIF(HAVE_STATIC_LIBGCC_FLAG)
-		check_cxx_compiler_flag (-static-libstdc++ HAVE_STATIC_LIBSTDCXX_FLAG)
-		IF(HAVE_STATIC_LIBSTDCXX_FLAG)
-			SET(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -static-libstdc++")
-		ENDIF(HAVE_STATIC_LIBSTDCXX_FLAG)
+		SET(CMAKE_EXE_LINKER_FLAGS "-static-libgcc -static-libstdc++")
+		#check_cxx_compiler_flag (-static-libgcc HAVE_STATIC_LIBGCC_FLAG)
```

Comment:
Old habits die hard? I'll remove it, I'm currently working on getting
the linux release prefix back in.

On 7/17/15, BevapDin <notifications@github.com> wrote:
>> @@ -88,14 +89,15 @@ IF (NOT DYNAMIC_LINKING)
>>  ELSE (NOT DYNAMIC_LINKING)
>>  	IF(MINGW)
>>  		# Avoid depending on MinGW runtime DLLs
>> -		check_cxx_compiler_flag (-static-libgcc HAVE_STATIC_LIBGCC_FLAG)
>> -		IF(HAVE_STATIC_LIBGCC_FLAG)
>> -			SET(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS}
>> -static-libgcc")
>> -		ENDIF(HAVE_STATIC_LIBGCC_FLAG)
>> -		check_cxx_compiler_flag (-static-libstdc++ HAVE_STATIC_LIBSTDCXX_FLAG)
>> -		IF(HAVE_STATIC_LIBSTDCXX_FLAG)
>> -			SET(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS}
>> -static-libstdc++")
>> -		ENDIF(HAVE_STATIC_LIBSTDCXX_FLAG)
>> +		SET(CMAKE_EXE_LINKER_FLAGS "-static-libgcc -static-libstdc++")
>> +		#check_cxx_compiler_flag (-static-libgcc HAVE_STATIC_LIBGCC_FLAG)
>
> What's it with all the commented code? If it isn't needed anymore, just
> remove it. It already exists in git so it can be viewed and restored
> anytime.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/CleverRaven/Cataclysm-DDA/pull/12946/files#r34929664